src/navigator/tab-bar/tab-bar.tsx

```diff
@@ -32,8 +33,12 @@ export const TabBar: FC = () => {
 
   const isHidden = isDefined(currentRouteName) && screensWithoutTabBar.includes(currentRouteName);
 
+  const isAndroid11Plus = isAndroid && platformVersion > ANDROID_10_VERSION;
+  const statusBarHeight = StatusBar.currentHeight ?? 0;
+
+  const androidBottomInset = isAndroid11Plus ? statusBarHeight : 0;
```

Comment:
This logic may need to be extended for iOS. Then this variable should be renamed